src/Utilities/Workloads.js

```diff
@@ -144,3 +145,48 @@ export const pruneWorkloadsRulesFilters = (localFilters, filterCategories) => {
     return arr;
   }, []);
 };
+
+export const switchSort = (sortIndex, item) => {
+  const rule = item[0]?.rule; // Using optional chaining to check if 'rule' is defined
+  switch (sortIndex) {
+    case 1:
+      return rule?.details || '';
+    case 2:
+      return rule?.total_risk || [];
```

Comment:
Why [] was selected as a default value for the number-type value?

Reply:
Mistake, updated it to a string